pycmr/granule.py

```diff
@@ -17,6 +17,12 @@ def __init__(self):
         self.params = {}
         self.options = {}
 
+    def _urlEncodeString(self, input):
```

Comment:
Might as well leave this for a future update, has no impact if it's left there for now.